Makefile

```diff
@@ -135,11 +135,9 @@ goveralls: $(GOVERALLS_BIN)
 	$(GOVERALLS_BIN) -coverprofile=coverage.txt -service=travis-ci
 
 
-travis-race: lint btcd unit-race
+travis-race: btcd unit-race
 
-travis-cover: lint btcd unit-cover goveralls
-
-travis-itest: lint itest
+travis-cover: btcd unit-cover goveralls
```

Comment:
No longer need btcd here either since it's done in the build stage?

Reply:
i thought about removing it, but idt the binary itself is cached. so i think it safe to leave it. it should insta-compile tho since the cache is warm bc of build stage